### ISSUE TITLE
Improve default colour map selection logic

### ISSUE THREAD

**Issue by htmlboss** [DFO-Ocean-Navigator/Ocean-Data-Map-Project#597]
At the moment we hard code each variable key and a corresponding default colour map. While this is ok, as we're adding new variables through datasets and calculations, things can get a little crazy with keeping the dictionary up to date, We should look at using some sort of regex pattern or whatever to give a good first-guess for a default. We can still hard code stuff as needed but at least we wouldn't have to babysit it.

**Comment by NoahGallant-MUN** [comment]
I agree, I'm not sure a regex pattern would work though since the variable types using the same colourmap could be vastly different.
A datasetconfig entry could work, but I'm hesitant to continuously add more and more to that.
